yb-voyager/cmd/importData.go

```diff
@@ -883,7 +883,12 @@ func doOneImport(task *SplitFileImportTask, connPool *tgtdb.ConnectionPool) {
 
 			log.Infof("%q => %d rows affected", copyCommand, rowsAffected)
 			if copyErr != nil {
-				utils.ErrExit("COPY %q FROM file %q: %s", task.TableName, inProgressFilePath, copyErr)
+				if strings.Contains(copyErr.Error(), "violates unique constraint") {
+					rowsAffected = task.OffsetEnd - task.OffsetStart
```

Comment:
If some of the rows from the batch were deleted, then this is going to report "extra" rows imported.

Reply:
yeah right, it should also be:
```
rowsAffected = reader.NumRecords
```

Good catch

Reply:
@amit-yb yeah right, Good catch
We can't go with this, because error must interrupted the reader execution:
```
rowsAffect = rr.NumRecords
```

Only approach in that i think will be do something like `wc -l` in this scenario. 
Let me know what you suggest.